iron-core/src/main/java/io/axway/iron/core/spi/file/AtomicBigInteger.java

```diff
@@ -0,0 +1,27 @@
+package io.axway.iron.core.spi.file;
```

Comment:
instead of creating this class, i would the internal state of file SPI with longs and adapt to BigInteger only when required by SPI API. WDYT ?

Reply:
I would like to able to use FileTransactionStore with AwsS3SnapshotStore with BigInteger to test AwsS3SnapshotStore (without KinesisTransactionStore), what impacts FILENAME_FORMAT, FILENAME_PATTERN, m_tmpCounter, m_nextTxId, m_consumerNextTxId. I think that using long internally is not possible to support that.
What I could do is making FileTransactionStore generic and providing two "flavors", Long and BigInteger.

Reply:
Add generic will pollute all the code, probably a bad idea.